app/components/provider_interface/find_candidates/filters_component.rb

```diff
@@ -77,6 +77,14 @@ def course_type_options
         end
       end
 
+      def path_to_clear_filters
+        if filter.is_a?(ProviderInterface::NotSeenCandidatesFilter)
+          provider_interface_candidate_pool_not_seen_index_path(remove_filters: true)
+        else
+          provider_interface_candidate_pool_root_path(remove_filters: true)
+        end
```

Comment:
```suggestion
        "?remove_filters=true"
```

This will mean that the link will point to whatever the current URL without having to do any logic 👌